### PR TITLE
Modify cause msg for pr and push events

### PR DESCRIPTION
**Pull request by minz1027**
- Modify cause msg for pr and push events

### CODE REVIEW

**Comment by petey** [comment]
What I REALLY want to display is the PR title and optionally have the ability to display the PR body. The users are still not going to know what `opened/created/merged by batman` has anything to do with a given sha.

**Comment by tkyi** [comment]
@petey we have `commit.message` right now
https://api.screwdriver.cd/v4/pipelines/613a05120b3c7166749d049b549d5edece076cdf/events
Although most of them start with "Merged pull request.."

File: plugins/webhooks/index.js [hunk 1]
```diff
@@ -78,7 +78,8 @@ function startPRJob(options, request) {
                 type: 'pr',
                 workflow: [job.name],
                 username,
-                sha
+                sha,
+                causeMessage: `${options.action} by ${username}`
```
Comment (d2lam) [inline plugins/webhooks/index.js]: I think it will be more clear to add `Pull request` in front of this: `Pull request synchronized by ... `

Comment (tkyi) [inline plugins/webhooks/index.js]: We would have to click the PR-## link to get to the PR build page, so I don't think it's necessary to add `Pull request` to the cause message

**Pull request merged**